cnf/meson.build

```diff
@@ -0,0 +1,133 @@
+install_data(
+    [
```

Comment:
I'm not sure what you mean by this. The first thing that install_data does when receiving a "list of strings or list of files() objects" parameter is internally convert all bare strings into files() objects using the same code that files() uses to check if they exist up front.

Are you aware of cases where this does not happen? If so, then it is a meson bug, because the configure stage should *never* succeed when the build or install stages can be trivially detected as impossible to succeed, such as for example when rule inputs are missing.

The difference between using strings and using files() objects is supposed to be limited to the fact that files() can be defined in one subdir and used in another subdir while remembering which subdir they were defined in (and therefore always resolving relative to the subdir they were defined in rather than the subdir they were used in).